server/views/pages/overview.njk

```diff
@@ -329,7 +329,7 @@
                   <li>{{ condition.text }}
                     {% if condition.image %}
                       <a
-                          href="/licence-image/?licenceId={{ licenceConditions.licenceId }}&conditionId={{ condition.id }}&prisonerNumber={{ risonerNumber }}"
```

Comment:
I think there might be an additional issue with the controller after you click on the link as it should redirect to the error page rather than render an error within the page.

Also could you add some unit tests and cypress tests around this?